index.html

```diff
@@ -33,5 +33,183 @@ <h2 class="social">LET'S CONNECT</h2>
+                </div>
+    
+                <div class="card">
+                    <img src="./img/Snapshoot Portfolio (2).png" alt="snapshot poortfolio" class="snapshot">
+                    <div class="works-description">
+                        <div class="works-intro">
+                            <h1 class="project-title">Tonic</h1>
+                            <div class="frame">
+                                <span class="client">
+                                    CANOPY</span>
+                                <img src="./img/Counter.png" alt="" class="counter">
+                                <span class="role">Back End Dev</span>
+                                <img src="./img/Counter.png" alt="" class="counter">
+                                <span class="year">2015</span>
+                            </div>
+                        </div>
+                    </div>
+                    <span class="details">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </span>
+                    <ul class="tags">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button class="action-button">See Project</button>
+                </div>
+    
+                <div class="card">
+                    <img src="./img/Snapshoot Portfolio (3).png" alt="snapshot poortfolio" class="snapshot">
+                    <div class="works-description">
+                        <div class="works-intro">
+                            <h1 class="project-title">Multi-Post
+                                Stories</h1>
+                            <div class="frame">
+                                <span class="client">
+                                    CANOPY</span>
+                                <img src="./img/Counter.png" alt="" class="counter">
+                                <span class="role">Back End Dev</span>
+                                <img src="./img/Counter.png" alt="" class="counter">
+                                <span class="year">2015</span>
+                            </div>
+                        </div>
+                    </div>
+                    <span class="details">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </span>
+                    <ul class="tags">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button class="action-button">See Project</button>
+                </div>
+          </div>
+        </section>
+
+        <section class="about">
+            <div class="intro">
+                <span class="title">About Myself</span>
```

Comment:
- [ ] Kindly replace this `span` tag with an `hx` tag (`h2` for example) since this is a heading and it would be semantically correct 👍